tools/run_tests/xds_k8s_test_driver/framework/xds_url_map_test_resources.py

```diff
@@ -266,11 +266,10 @@ def cleanup(self) -> None:
     @functools.lru_cache(None)
     def default_backend_service(self) -> str:
         """Returns default backend service URL."""
-        self.td.load_backend_service()
-        return self.td.backend_service.url
+        return self.td.make_resource_name(self.td.BACKEND_SERVICE_NAME)
```

Comment:
This seems to break the tests?
This function is expected to return the URL
- `projects/${PROJECT_ID}/global/backendServices/backend-service`, 
- or the version with `https://www.googleapis.com/compute/v1/` prefix

But now it returns just the name `backend-service`, and the URL-map creation will fail.

Is that right?

Reply:
You are right. Created another fix attempt at https://github.com/grpc/grpc/pull/26748.